### PR TITLE
feat(dotnet): WPF and WinForms

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: src/collections/_documentation/platforms/dotnet/winforms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+sidebar_order: 30
+---
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WinForms specific configuration.
+
+Sentry's .NET SDK works with WinForms applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WinForms apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
+
+## Configuration
+
+The SDK automatically captures unhandled exceptions that hit the `AppDomain.UnhandledException`, but on Windows Forms, for production apps (when no debugger is attached), the small Window pops up when a crash happens. In order to get the exception to rethrow and be captured by Sentry, you must also configure:
```
Comment (PeloWriter) [inline src/collections/_documentation/platforms/dotnet/winforms.md:12]: suggest: "The SDK automatically handles `AppDomain.UnhandledException`. On Windows Forms, for production apps (when no debugger is attached), the small Window pops up when an unhandled error occurs. To get the exception to rethrow the error so it is captured by Sentry, also configure:"

File: src/collections/_documentation/platforms/dotnet/wpf.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Windows Presentation Foundation
+sidebar_order: 40
+---
+
+Sentry's .NET SDK works with Windows Presentation Foundation applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WPF apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
+
+## Configuration
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WPF specific configuration.
```
Comment (PeloWriter) [inline src/collections/_documentation/platforms/dotnet/wpf.md:10]: Suggest (similar to previous edit): "In addition to [configuring the main call SentrySdk.Init](/platforms/dotnet/), you must also configure the WPF specific configuration"

File: src/collections/_documentation/platforms/dotnet/wpf.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+The SDK automatically captures unhandled exceptions that hit the `AppDomain.UnhandledException`, but on WPF, for production apps (when no debugger is attached), WPF catches exception to show the dialog to the user. You can also configure your app to capture those exceptions before the dialog shows up:
```
Comment (PeloWriter) [inline src/collections/_documentation/platforms/dotnet/wpf.md:12]: suggest: "The SDK automatically handles AppDomain.UnhandledException. On WPF, for production apps (when no debugger is attached), WPF captures an exception to show dialog to the user."

File: src/collections/_documentation/platforms/dotnet/winforms.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Windows Forms
+sidebar_order: 30
+---
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WinForms specific configuration.
+
+Sentry's .NET SDK works with WinForms applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WinForms apps running on .NET Framework 4.6.1, .NET Core 3.0 or higher.
```
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/dotnet/winforms.md:8]:
```suggestion
Sentry's .NET SDK works with WinForms applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WinForms apps running on .NET Framework 4.6.1, .NET Core 3.0, or higher.
```

File: src/collections/_documentation/platforms/dotnet/winforms.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+sidebar_order: 30
+---
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WinForms specific configuration.
+
+Sentry's .NET SDK works with WinForms applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry). It works with WinForms apps running on .NET Framework 4.6.1, .NET Core 3.0, or higher.
+
+## Configuration
+
+The SDK automatically captures unhandled exceptions that hit the `AppDomain.UnhandledException`, but on Windows Forms, for production apps (when no debugger is attached), the small Window pops up when a crash happens. In order to get the exception to rethrow and be captured by Sentry, you must also configure:
```
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/dotnet/winforms.md:12]:
```suggestion
The SDK automatically handles AppDomain.UnhandledException. On Windows Forms, for production apps (when no debugger is attached), the small Window pops up when an unhandled error occurs. To get the exception to rethrow the error so it is captured by Sentry, also configure:
```

File: src/collections/_documentation/platforms/dotnet/winforms.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+title: Windows Forms
+sidebar_order: 30
+---
+
+Besides the [main configuration like calling `SentrySdk.Init`](/platforms/dotnet/), you must consider the WinForms specific configuration.
```
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/dotnet/winforms.md:6]:
```suggestion
In addition to initializing the SDK with `SentrySdk.Init`](/platforms/dotnet/), you must also configure WinForms to error handler.
```

File: src/collections/_documentation/platforms/dotnet/wpf.md (hunk 3 above)
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/dotnet/wpf.md:12]:
```suggestion
The SDK automatically handles `AppDomain.UnhandledException`. On WPF, for production apps (when no debugger is attached), WPF catches exception to show the dialog to the user. You can also configure your app to capture those exceptions before the dialog shows up:
```

File: src/collections/_documentation/platforms/dotnet/wpf.md (hunk 2 above)
Comment (bruno-garcia) [inline src/collections/_documentation/platforms/dotnet/wpf.md:10]:
```suggestion
In addition to [initializing the SDK with `SentrySdk.Init`](/platforms/dotnet/), you must configure the WPF specific error handler.
```

**Pull request merged**